dear_petition/petition/utils.py

```diff
@@ -0,0 +1,16 @@
+import pytz
+from datetime import datetime, date
+
+
+def dt_obj_to_date(dt_obj):
+    """converts a datetime obj to a date"""
+    return dt_obj.date()
+
+
+def make_date_obj_aware(dt_obj):
+    """makes datetime obj aware of timezone (EST)"""
+    datetime_obj = datetime.strptime(dt_obj, "%Y-%m-%dT%H:%M:%S")
```

Comment:
Django has a [`make_aware()`](https://docs.djangoproject.com/en/dev/ref/utils/#django.utils.timezone.make_aware) utility function for this, which makes a datetime aware of the project's timezone, which can be set in `settings.TIME_ZONE`

Reply:
done `settings.TIME_ZONE='America/New York'`